config/vendor_api/draft.yml

```diff
@@ -192,6 +195,8 @@ paths:
       summary: Create interview
       description: |
         Creates a new interview for this application.
+
+        This will transition the application to the [interviewing state](/api-docs/lifecycle#interviewing) if it is not already in this state.
```

Comment:
we don't surface interviewing state in the API, so might be confusing, I suggest we leave it out